Makefile

```diff
@@ -153,38 +153,47 @@ else
 endif
 
 test-cli: PYTEST_MARKER=category_cli and (not flaky)
+test-cli: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=upb
```

Comment:
Your PR description mentions a different environment variable value `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python`, typo?

Reply:
Not exactly; we are obliged to use slow Python implementation only when we work with TensorFlow-hub, which happens only in test-nlu-featurizers. We can use fast C implementation for all other cases, which is upb.